edx_sga/static/css/edx_sga.css

```diff
@@ -45,25 +45,36 @@
 
 .sga-block .upload {
     position: relative;
+    height: 40px;
 }
 
-.sga-block .upload button {
+.sga-block .upload label {
```

Comment:
there is style issue.
<img width="459" alt="screen shot 2017-09-06 at 1 49 53 pm" src="https://user-images.githubusercontent.com/10431250/30102841-49c699ac-930a-11e7-85d4-723743c260b5.png">

Reply:
Thank you @amir-qayyum-khan  
I've changed these lines because of a bug in the xblock main page so I can't revert.
but I made some improvements to solve this issue.